### ISSUE TITLE
Ethernet drivers phy support on Live Kit initrd is missing

### ISSUE THREAD

**Issue by innovarew** [Tomas-M/linux-live#217]
TLDR; Mailed a description of the issue at the slax-users group, before learning that linux-live-kit development was on github, as this affects linux-live/slax existing releases, I think its relevant having this cross posting. 

https://groups.google.com/g/slax-users/c/qN2PTx8ftq8

## Problem

Basically what happens is:

Some Ethernet drivers, such as the ubiquitous Realtek 8139too.ko depend on net/phy/realtek.ko for the Ethernet PHY layer, as the net/phy is not included in  the initrd, those Ethernet drivers do not work.

## Fix

The fix is simply including the net/phy in the initrd

~~~
diff -Nur linux-live-2.6.orig/initramfs/initramfs_create linux-live-2.6/initramfs/initramfs_create
--- linux-live-2.6.orig/initramfs/initramfs_create      2022-05-20 09:41:52.390045698 +0200
+++ linux-live-2.6/initramfs/initramfs_create   2022-05-20 09:38:27.024502132 +0200
@@ -119,6 +119,7 @@
 if [ "$NETWORK" = "true" ]; then
    # add all known ethernet drivers
    copy_including_deps /$LMK/kernel/drivers/net/ethernet
+   copy_including_deps /$LMK/kernel/drivers/net/phy
 fi
~~~

**Comment by innovarew** [comment]
Booted the latest Slax-15 from https://www.slax.org/download/slax-64bit-15.0.0.iso and noticed that now the drivers/net/phy/ is correctly populated. 
Therefore, the issue is solved in slack-15, and can be closed.

Thanks!

~~~
root@slax:~# cat /etc/slackware-version
Slackware 15.0
root@slax:~# ls /run/initramfs/lib/modules/5.15.19/kernel/drivers/net/phy/libphy.ko
/run/initramfs/lib/modules/5.15.19/kernel/drivers/net/phy/libphy.ko
~~~
